enhancements/console/quick-starts.md

```diff
@@ -212,6 +223,26 @@ metadata:
     console.openshift.io/name: explore-serverless
 ```
 
+#### Disabling Quick Starts
+
+`operator.openshift.io/v1` / `Console` CRD provides a `spec.customization` property where an admin will list the Quick Starts to be disabled.
+
+Console operator CRD with disabled Quick Starts:
```

Comment:
I assume this will disable quick starts with this `metadata.name` or `console.openshift.io/name` annotation (for translated quick starts). If so, I'd call that out specifically here.

Reply:
Good catch! Yes this will match the `console.openshift.io/name` annotation. It should only match against `metadata.name` if there is no annotation.

Reply:
done